src/style.css

```diff
@@ -62,6 +62,9 @@
   --color-heading: #1a202c;
   --color-heading-black: black;
   --color-accent: #d1dce5;
+  --color-white: #f7fafc;
+  --letter-spacing-2: 2px;
+  --letter-spacing-3: 7px;
```

Comment:
The second letter spacing I use is `4px`. This is the spacing in the `h3` header.